src/main/java/uk/gov/hmcts/reform/civil/service/InitiateGeneralApplicationService.java

```diff
@@ -293,10 +293,6 @@ public boolean respondentAssigned(CaseData caseData) {
         return true;
     }
 
-    public boolean isGAApplicantSameAsParentCaseClaimant(CaseData caseData, UserDetails userDetails) {
```

Comment:
I would suggest not removing this method because It will increase direct dependency on HelperClass from InitiateGeneralApplicationHandler. Helper class should be accessed service class. This is just for code organization.